src/vs/workbench/services/suggest/browser/simpleSuggestWidgetDetails.ts

```diff
@@ -72,6 +74,31 @@ export class SimpleSuggestDetailsWidget {
 		this._type = dom.append(this._header, dom.$('p.type'));
 
 		this._docs = dom.append(this._body, dom.$('p.docs'));
+
+		this._configureFont();
+
+		// TODO: respond to font config change?
```

Comment:
@tyriar, do you think it's overkill to add another property that gets passed in with the configuration keys that should be monitored or an event for when font changes?

Reply:
added